### PR TITLE
Docs: Add some first basic docs about building the docs with mr.docs

### PR DESCRIPTION
**Pull request by svx**
Some first basic docs, about how to build the docs of papyrus for testing

### CODE REVIEW

**Comment by svx** [comment]
Would someone so kind to review? IMHO it would be nice to get that merged :)

Thanks !

File: docs/building_docs.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+.. topic:: How to build the docs
+
+    Sometimes you want to build the docs of papyrus
+    for getting a visual how the would look like, or maybe
+    you want to build and deploy them.
+
+Documentation Setup
+===================
+
+Papyrus follows the *best practrices* of `Plone Documentation <http://docs.plone.org/about/documentation_styleguide.html>`_.
```
Comment (polyester) [inline docs/building_docs.rst]: typo... "practices"

File: docs/building_docs.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+All the docs are written in *rst* [reStructuredText] and inside the */docs* directory.
+
+If you add an document, please *do not forgett* to update the index [index.rst].
```
Comment (polyester) [inline docs/building_docs.rst]: typo "forget"

**Comment by polyester** [comment]
cannot check the _functionality_ now, have no docker on work machine.

**Comment by jensens** [comment]
There was no activity on this for a while. I close this request for now. If you plan to work further on this please reopen and go on.

**Pull request closed without merging**